cookbook/configuration/environments.rst

```diff
@@ -6,7 +6,7 @@ How to Master and Create new Environments
 
 Every application is the combination of code and a set of configuration that
 dictates how that code should function. The configuration may define the
-database being used, whether or not something should be cached, or how verbose
+database being used, if something should be cached or how verbose
```

Comment:
Ah, you're right. I misunderstood the sentence before (didn't read it carefully enough). Sorry for that.